### DISCUSSION TITLE
[求助]师太要收手机了怎办

### DISCUSSION THREAD

**Discussion by HeroBrine-c** [share121/inter-knot#4576] [委托]
![微信图片_20240908003700](https://github.com/user-attachments/assets/5dc5ca49-f483-48fa-86c6-0f28c23d161c)
# 丸辣

**Comment by HeroBrine-c** [comment]
。<img src="https://img-static.mihoyo.com/communityweb/upload/ad0c5f768b9022b8b9487c3dfb3c8b59.png" alt="倒" width="75" >

**Comment by loftTin** [comment]
从了吧

**Comment by KELERSAMA** [comment]
收手机的时候，酝酿一下情绪，搂住老师的腰跟老师说“beby 以后，你管手机，我管你哦↗↘”老师肯定会臣服，无非两种结局，要么老师不收你的手机 要么你可以获得老师的控制权解放全班手机

**Comment by SeeleVollereid** [comment]
给老师说你没带还有手机在家长那里保管(老师应该会同意不收)，老师总不可能到你家去收吧？(总之如果实在不行就交了吧)

**Reply by HeroBrine-c** [reply]
你怎么知道她上次跑到我家收了我手机(
![52DA4614](https://github.com/user-attachments/assets/d674c0a8-b13b-4b2c-bc67-a41b9250865d)
)

**Comment by mztxy229** [comment]
从了吧

**Comment by sensei-is-playing-GenShin** [comment]
我有一个方法，但还是在试验期，就是拿家里的旧手机或者上咸鱼买个2块钱的OPPO a5套上一个和你手机差不多样子的手机壳，然后糊弄过去

**Comment by MojangAB-G50** [comment]
我也是

**Comment by For-elysia** [comment]
花二十块去找“回收旧手机，破手机，不中用的手机”的爷爷奶奶叔叔阿姨们要一个，当然哈，虽然学校经常干些不是人的事儿，但不要影响学习哈

**Comment by MELTlilith520** [comment]
找个旧手机或者买个模型机之类的应该可以？手机不交没什么，但还是要认真学习哦
